### PR TITLE
Task s3 create products page

### PR DESCRIPTION
**Pull request by ailindoyle**
Product page template in HTML and CSS.

Index page refactored to match product page structure, as first version had some issues with responsiveness on class container.

Styles.css updated for refactored index.php and new product.php

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -4,9 +4,89 @@
...
+        <a href="#">
+            <h2>Home</h2>
+        </a>
+    </div>
+    <div class="shop-name col-xs-10">
+        <h2>Totally Pawesome</h2>
+    </div>
+</section>
+<section class="row main">
+    <div class="sidebar col-xs-2">
```
Comment (o-tahboub) [inline index.php]: Row responsiveness seems to have changed, the side bar is not large enough in smaller sizes.

File: index.php [hunk 2]
```diff
@@ -4,9 +4,89 @@
...
+        </a>
+        <a href="#">
+            <h4>Category5</h4>
+        </a>
+        <a href="#">
+            <h4>Category6</h4>
+        </a>
+    </div>
+    <div class="main-content col-xs-10">
+        <div class="row">
```
Comment (o-tahboub) [inline index.php]: Categories used to be 3 blocs across, this has changed to 2, might be an issue as the existing wireframe shows 3 across for categories.

File: index.php [hunk 3]
```diff
@@ -4,9 +4,89 @@
...
+                <a href="#">
+                    <img class="img-thumbnail" src="src/assets/img/cat1.jpg">
+                    <h4>Category 6</h4>
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
 </body>
+
```
Comment (o-tahboub) [inline index.php]: This is different from the file below, I think its better if you get rid of it.

File: src/app/products.php [hunk 4]
```diff
@@ -1 +1,138 @@
...
+        <a href="#">
+            <h4>Category6</h4>
+        </a>
+    </div>
+    <div class="main-content col-xs-10">
+        <div class="row">
+            <h2>Product Category</h2>
+        </div>
+        <div class="row">
+            <div class="product col-xs-8 col-sm-6 col-md-4 col-lg-3">
```
Comment (o-tahboub) [inline src/app/products.php]: These are showing 2 blocks across, the wireframe has 4 blocks across.

**Review by o-tahboub — COMMENTED** [review]
Please check comments.

File: index.php (hunk 2 above)
Comment (ailindoyle) [reply index.php]: categories are still 3 blocks across on medium screens +, are two bloc on small screens and one on extra small screens

File: src/app/products.php (hunk 4 above)
Comment (ailindoyle) [reply src/app/products.php]: these are 4 blocks across on large screens, 3 blocs across on medium, 2 blocks on small, and 1 on extra small - having more breaks helps with smooth responsiveness

File: index.php (hunk 1 above)
Comment (kyam-mayden) [reply index.php]: - to add, it may need a minimum width

File: src/app/products.php (hunk 4 above)
Comment (kyam-mayden) [reply src/app/products.php]: May be a local issue, mine is showing 4

File: index.php (hunk 3 above)
Comment (kyam-mayden) [inline index.php]: Whitespace

**Review by kyam-mayden — APPROVED** [review]

**Pull request merged**